packages/ui/src/theme/overwrites/components/input.ts

```diff
@@ -0,0 +1,32 @@
+export const Input = {
+  sizes: {
+    sm: {
+      field: {
+        padding: '6px 12px',
+        borderRadius: '6px',
```

Comment:
I think you can define `borderRadius` in the outline variant config to don't repeat yourself

Reply:
Yes, indeed 😅